toolkit/tools/pkg/imagecustomizerlib/grubcfgutils.go

```diff
@@ -591,6 +591,28 @@ func selinuxModeToArgs(selinuxMode imagecustomizerapi.SELinuxMode) ([]string, er
 	return newSELinuxArgs, nil
 }
 
+// Converts an SELinux mode into the list of required command-line args for that mode (with enforcing mode).
+func selinuxModeToArgsWithEnforcingArg(selinuxMode imagecustomizerapi.SELinuxMode) ([]string, error) {
+	newSELinuxArgs := []string(nil)
+	switch selinuxMode {
+	case imagecustomizerapi.SELinuxModeDisabled:
+		newSELinuxArgs = []string{installutils.CmdlineSELinuxDisabledArg}
+
+	case imagecustomizerapi.SELinuxModeForceEnforcing:
+		newSELinuxArgs = []string{installutils.CmdlineSELinuxSecurityArg, installutils.CmdlineSELinuxEnabledArg,
+			installutils.CmdlineSELinuxEnforcingArg}
+
+	case imagecustomizerapi.SELinuxModePermissive, imagecustomizerapi.SELinuxModeEnforcing:
```

Comment:
Would it make sense to create a new mode called `SELinuxModeForcePermissive` to MIC? That way, we can have a consistent API between MIC and EMU.

Reply:
I'm not sure if it makes sense to add a force-permissive mode in MIC, the trident will pass value `permissive` not `force-permissive` to EMU. The difference between EMU' permissive and MIC's permissive is EMU needs to add enforicing=0 and not modify etc/selinux/config